packages/components/src/components/hds/badge-count/index.ts

```diff
@@ -6,14 +6,35 @@
 import Component from '@glimmer/component';
 import { assert } from '@ember/debug';
 
-export const DEFAULT_SIZE = 'medium';
-export const DEFAULT_TYPE = 'filled';
-export const DEFAULT_COLOR = 'neutral';
-export const SIZES = ['small', 'medium', 'large'];
-export const TYPES = ['filled', 'inverted', 'outlined'];
-export const COLORS = ['neutral', 'neutral-dark-mode'];
-
-export default class HdsBadgeCountIndexComponent extends Component {
+import {
+  HdsBadgeCountSizeValues,
+  HdsBadgeCountTypeValues,
+  HdsBadgeCountColorValues,
+} from './types.ts';
+import type {
+  HdsBadgeSizes,
+  HdsBadgeTypes,
+  HdsBadgeCountColors,
+} from './types.ts';
```

Comment:
Personally not as opinionated on this, but I remember you were keen on [getting these ordered alphabetically](https://github.com/hashicorp/design-system/pull/1991#discussion_r1524606556).

Reply:
Let's add it to the list of things we want to standardize across components (eg. order of the Args: "logical"-ish, like it appears in the Component API docs, or in the getters order, or more simply alphabetical?)

Reply:
For now I'll sort these alphabetically.